src/librustc/lib.rs

```diff
@@ -75,6 +75,7 @@
 #![feature(trusted_len)]
 #![feature(catch_expr)]
 #![feature(test)]
+#![feature(inclusive_range)]
```

Comment:
Should these be `cfg_attr` with stage0?

Reply:
Probably, I didn't see any errors about that though